packages/test/test-service-load/src/optionsMatrix.ts

```diff
@@ -50,6 +50,7 @@ export function generateRuntimeOptions(seed: number) {
     const runtimeOptionsMatrix: OptionsMatrix<IContainerRuntimeOptions> = {
         gcOptions: [undefined, ...generatePairwiseOptions(gcOptionsMatrix, seed)],
         summaryOptions: [undefined, ...generatePairwiseOptions(summaryOptionsMatrix, seed)],
+        loadSequenceNumberVerification: [undefined],
```

Comment:
We can add the expected values here right? I don't see any harm in testing these. Infact, stress tests are what found this issue in the first place so might be worth adding coverage.

Reply:
We can, but I'm not sure that it adds much value/coverage, since it just means testing this small if statement of what to do depending on that value.

I assume the cost of adding various options to this matrix is multiplicative? i.e. I think it's really important and good to add all possible runtime options here, but knowing that the cost is nontrivial- and perhaps choosing not to for such a few lines of code & impact. @anthony-murphy thoughts? I could be wrong about the cost, but definitely it is either impacting performance of these tests _or_ decreasing number of unique cases.